man/loader.conf.xml

```diff
@@ -64,14 +64,13 @@
       <varlistentry>
         <term>timeout</term>
 
-        <listitem><para>How long the boot menu should be shown before the default
-        entry is booted, in seconds. This may be changed in the boot menu itself and
-        will be stored as an EFI variable in that case, overriding this option.
+        <listitem><para>How long the boot menu should be shown before the default entry is booted (in seconds,
+        maximum is 60) or <literal>off</literal>. This may be changed in the boot menu itself and will be
```

Comment:
Limiting the timeout to 60 seconds is bad. A user may set the timeout to 5 minutes quite resonable, for example to be able to have a finite timeout but to still be able to get coffee while rebooting.

Reply:
I guess it is very subjective what a reasonable timeout is and hence I agree that this whole pull request is likely going in the wrong direction.

Reply:
This sounds very sloppy. Note that the EFI variable for the timeout is actually just a string. I'd thus just say, let's define a new value "infinity" or so that can be stored there. And when it is read it's returned as UINTN_MAX...

Some arbitrary threshold sounds wrong to me.

Reply:
Agreed. However I have concerns about this approach as it overwrites the value. See this [comment](https://github.com/systemd/systemd/pull/16320#issuecomment-652333110) for details.